packages/components/src/components/text-field/text-field.tsx

```diff
@@ -105,6 +105,8 @@ export class TextField {
   @Prop() list?: string;
   /** (optional) the input should automatically get focus when the page loads. */
   @Prop() inputAutofocus?: boolean;
+  /** (optional) custom value for automplete HTML attribute */
+  @Prop() autocomplete?: string;
```

Comment:
can you rename it to `inputAutocomplete`, please? So it won't clash with the native attribute name.

Reply:
Thanks for your feedback, I changed/add it in my last [commit](https://github.com/telekom/scale/pull/1400/commits/ea1b4a868bc70ad7ae6b0bd0863643a658967ccc).